### DISCUSSION TITLE
Blocklists. Active or not?

### DISCUSSION THREAD

**Discussion by squromiv** [TechnitiumSoftware/DnsServer#445] [Q&A] [answered]
I do not exactly understand how this feature works (Windows install). For example:
1) I add two blocklist urls from default list (Settings->Blocking)
![screen_2022 08 05](https://user-images.githubusercontent.com/15985877/183026688-badd7703-b882-4596-95e4-1b8e453aae90.jpg)
Click "Update now" button->Click "Save settings"
So there are two new files in this folder now
c:\Program Files (x86)\Technitium\DNS Server\config\blocklists
![screen_2022 08 05 002_2](https://user-images.githubusercontent.com/15985877/183028028-29bb39c3-8e2c-41e1-a99d-93ed85a54dfb.jpg)

2) I use these blocklists for some time and then decide to apply new blocklist instead of them.
Delete strings->Add new blocklist->Click "Update now" button->Click "Save settings"
![screen_2022 08 05_3](https://user-images.githubusercontent.com/15985877/183032249-def4227f-835b-4979-8bba-e81ca2fcfc21.jpg)

So there are three files in the folder now
![screen_2022 08 05 002_4](https://user-images.githubusercontent.com/15985877/183029391-84625c4f-1ddb-45c2-911c-a4a06986747e.jpg)

And how to clear what blocklist is in use now? This one, that set in "Block List URLs" dashboard? Or those ones, which are placed in the folder config\blocklists ? It may be stupid proposal, but I would like to see something like checkboxes in this settings area. Also what is the reason to store "old" blocklists, which are not in use now?

**Comment by ShreyasZare** [comment, marked as answer]
Thanks for asking. The settings that you see in the GUI is what is running. The files that you see on disk are just cached files.

**Comment by squromiv** [comment]
Great software and support!

Something like this is more familiar to me. Checkboxes and buttons to add, edit and delete unnecessary files.
![screen_2022 08 05 002_5](https://user-images.githubusercontent.com/15985877/183039175-19823932-b603-4c4a-9462-34e67f872b45.jpg)

**Reply by ShreyasZare** [reply]
Yes, something like that can be done. But it will take time since core DNS features are given priority over GUI features when deciding on the next update.

**Reply by squromiv** [reply]
> Yes, something like that can be done. But it will take time

Thanks a lot. Do as you do. That is just my thought how to make this app more user-friendly.
